src/app/Shared/Services/CommandChannel.service.tsx

```diff
@@ -34,25 +35,22 @@ export class CommandChannel {
         (err: any) => this.logError('Grafana Datasource configuration', err)
       );
 
-    this.onResponse('disconnect').pipe(
-      filter(msg => msg.status !== 0),
-    ).subscribe(msg => this.logError('Command failure', msg));
+    fromFetch(`${this.apiSvc.authority}/api/v1/grafana_dashboard_url`)
```

Comment:
When I view file, it looks like this fromFetch is duplicated from above; likely not intended?

Reply:
One is dashboard and one is datasource ;-)

Reply:
Oh, you're right. My mistake, sorry.